src/main/kotlin/screepsai/roles/Role.kt

```diff
@@ -33,11 +40,12 @@ abstract class Role(val creep: Creep) {
          */
         fun build(creepRole: CreepRole, creep: Creep): Role {
             return when (creepRole) {
-                CreepRole.UNASSIGNED -> Harvester(creep)
-                CreepRole.HARVESTER -> Harvester(creep)
-                CreepRole.UPGRADER -> Upgrader(creep)
+                CreepRole.UNASSIGNED  -> Harvester(creep)
```

Comment:
The harvester role is really specific. Probably should have this default to something else instead.

However, the unassigned role should be an impossible case to run into.

Could make an `Unassigned` class with some general purpose logic and a general purpose logic that could be useful when bootstrapping a room or tackling manually started projects?